### ISSUE TITLE
kindle oasis3 can't display some files

### ISSUE THREAD

**Issue by ystyle** [leotaku/mobi#2]
After I generate azw3 with chaptered, when testing on kinlde oasis3, some files cannot be displayed on the device's library.

After testing, it was found that the first book of the chapters could not be displayed, and the subsequent second and third books could be displayed. I've tried changing the filename and the author has nothing to do with it. Tried reducing the number of chapters (200) and found that also only some of the files could be displayed.

In the attachment, file-1 cannot be displayed, and file-2 is normal

[samples.zip](https://github.com/leotaku/mobi/files/8012755/samples.zip)

**Comment by leotaku** [comment]
This seems to be an issue that arises when a chapter, by coincidence, ends/starts exactly on a chunk border. I suspect this because the first file you have provided reports incorrect trailing byte sequences only in chapters with this property.

I'll try to implement a fix and ping you again when I've pushed it, so you can then test if this resolves your issue.

**Comment by ystyle** [comment]
thanks

**Comment by leotaku** [comment]
@ystyle ping

In case the new changes on master did not fix the issue, please provide a broken ebook generated with the new version of the library and ideally also the code used to generate it.

**Comment by ystyle** [comment]
In the attachment, file-1 cannot be displayed, and file-2、file-3 is normal
[samples.zip](https://github.com/leotaku/mobi/files/8015139/samples.zip)

**Comment by ystyle** [comment]
> In the attachment, file-1 cannot be displayed, and file-2、file-3 is normal [samples.zip](https://github.com/leotaku/mobi/files/8015139/samples.zip)

After testing, file-3 is also not displayed

**Comment by leotaku** [comment]
Right, in that case, I don't currently understand what could be the cause of this.

Unfortunately, I also don't currently have access to my own Kindle Oasis reader and won't for another 1-2 weeks, so I can't really debug the problem, which you say only appears on the Kindle Oasis 3.

Have you tested the generated e-books on other Kindle devices, or is the Oasis 3 the only device you are using?

**Comment by ystyle** [comment]
I only have ko3, but I found someone to test it, and all three files can be displayed on kpw3 5.9.6.1
